lib/auth/grpcserver.go

```diff
@@ -1556,9 +1556,9 @@ func downgradeRole(ctx context.Context, role *types.RoleV5) (*types.RoleV5, erro
 		}
 	}
 
-	minSupportedVersionForV4Roles := semver.New(utils.VersionBeforeAlpha("6.2.4"))
-	if clientVersion == nil || clientVersion.LessThan(*minSupportedVersionForV4Roles) {
-		log.Debugf(`Client version "%s" is unknown or less than 6.2.4, converting role to v3`, clientVersionString)
+	minSupportedVersionForV5Roles := semver.New(utils.VersionBeforeAlpha("8.2.0"))
```

Comment:
We might be aiming to get this in 8.3.0 now, just FYI.

Reply:
Updated, I'll double check before I merge regardless.